lab-taylor/test/order-route-test.js

```diff
@@ -0,0 +1,185 @@
+'use strict';
```

Comment:
Great job!  You'll need to add tests for the DELETE route.  Additionally, instead of creating a new order for each route test it would be simpler to instead create a variable inside the outermost describe block and assign it the note created during the 200 POST route test.  In your other tests you should be able to just reference that order instead of creating new ones each time.  When you implement your DELETE route test, delete that order as the last test.  This way each test run will get a fresh start and you'll minimize the number of asynchronous requests you'll need to make throughout your tests.